charts/thoras/templates/api-server-v2/deployment.yaml

```diff
@@ -59,6 +59,8 @@ spec:
             value: "{{ .Values.thorasApiServerV2.slackErrorsEnabled | default .Values.slackErrorsEnabled }}"
           - name: "SERVICE_LOGLEVEL"
             value: {{ default .Values.logLevel .Values.thorasApiServerV2.logLevel }}
+          - name: "SERVICE_PROMETHEUS_API_URL"
+            value: {{ .Values.thorasReasoningApi.connectors.prometheus.baseUrl }}
```

Comment:
Is it okay to refer to the `thorasReasoningApi` values from other deployments? Or should I be specifying this value separately for the API server layer, even though it should point to the same thing?

Reply:
Or maybe the prometheus endpoint URL should be a secret that both can refer to?